src/renderer/components/+workloads-pods/pod-details-container.tsx

```diff
@@ -86,6 +87,9 @@ export class PodDetailsContainer extends React.Component<Props> {
         <DrawerItem name="Image">
           {image}
         </DrawerItem>
+        <DrawerItem name="ImageId">
+          {imageId}
+        </DrawerItem>
```

Comment:
Given that not everyone will need this maybe only have it as a tooltip, wdyt @aleksfront ?

```suggestion
        <DrawerItem name="Image" tooltip={imageId}>
          {image}
        </DrawerItem>
```

BTW, for the above suggestion to work you need to add `, TooltipDecoratorProps` to the `interface DrawerItemProps` and an `@withTooltip` line above the `class DrawerItem` statement.

Reply:
The question here would users need to copy `ImageId`? If we'll go with tooltip, we can do this by wrapping image value with `<Badge/>`:
```
<DrawerItem name="Image">
  <Badge label={image} tooltip={imageId}/>
</DrawerItem>
```

Reply:
You can copy from a badge so yes I think that is a better idea.